Example/SSDynamicTextExample/SSViewController.m

```diff
@@ -9,68 +9,140 @@
 #import "SSViewController.h"
 #import <SSDynamicText.h>
 
+static NSString * const kLabelText = @"This label, text field, and text view all support custom fonts and respond to changes in preferred text size.\n\nSwitch to Settings.app, change your preferred text size, then switch back here.";
+
 @interface SSViewController ()
 
 @property (nonatomic, strong) SSDynamicLabel *label;
 @property (nonatomic, strong) SSDynamicButton *button;
 @property (nonatomic, strong) SSDynamicTextField *textField;
 @property (nonatomic, strong) SSDynamicTextView *textView;
+@property (nonatomic, assign) BOOL isAttributedTextEnabled;
 
 @end
 
 @implementation SSViewController
 
+- (void)changeTexts:(UIButton *)sender {
+    if (self.isAttributedTextEnabled) {
+        [self setUpNormalTexts];
+    } else {
+        [self setUpAttributedTexts];
+    }
+    self.isAttributedTextEnabled = !self.isAttributedTextEnabled;
```

Comment:
Maybe move this line before `if ()`, because right now it looks:
attributedTextIsEnabled so set up normal text...